frontend/src/libs/components/navigation-menu-wrapper/libs/constants.ts

```diff
@@ -1,12 +1,25 @@
-import { AppRoute } from '#libs/enums/enums.js';
+import { AppRoute, SidebarModeQueryStringValue } from '#libs/enums/enums.js';
+import { getUrlWithQueryString } from '#libs/helpers/helpers.js';
 import { type Route } from '#libs/types/types.js';
 
+const urlWithQueryString = getUrlWithQueryString({
+  sidebarMode: SidebarModeQueryStringValue.SHOW,
+});
+
 const SIDEBAR_ROUTES: Route[] = [
   { path: AppRoute.ROOT, name: 'home', icon: 'home' },
-  { path: AppRoute.MEDITATION, name: 'meditation', icon: 'meditation' },
-  { path: AppRoute.CHATS, name: 'chats', icon: 'chats' },
-  { path: AppRoute.JOURNAL, name: 'journal', icon: 'journal' },
-  { path: AppRoute.PROFILE, name: 'profile', icon: 'user' },
+  {
+    path: urlWithQueryString(AppRoute.MEDITATION),
```

Comment:
&Can't we handle it via callback of setSidebarMode?

Reply:
but setSidebarMode is used in useSidebarState, and we can use hooks only in components. I didn't get what you mean

Reply:
moreover, the logic of the hook is focused on button clicks. not on links